crates/solvers/src/tests/legacy/jit_order.rs

```diff
@@ -0,0 +1,118 @@
+use {
```

Comment:
I really like the doc comments you have for other tests, I think they're a great idea. Can you add one here as well?